backends/p4tools/modules/testgen/targets/bmv2/test/P4Tests.cmake

```diff
@@ -3,7 +3,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/../../../cmake/TestUtils.cmake)
 # Add v1model tests from the p4c submodule
 set(P4TOOLS_BINARY_DIR ${CMAKE_SOURCE_DIR}/build)
 set(P4TESTGEN_DIR ${CMAKE_SOURCE_DIR}/build/testgen)
-set(P4TESTGEN_DRIVER "${P4TOOLS_BINARY_DIR}/p4check testgen")
```

Comment:
So `p4check` doesn't exist at all now. Any module needs to be invoked via it's name like `p4testgen`?

Reply:
Yes, after this PR `p4check` does not exist any more. All tests are run with the `p4testgen` binary.